### DISCUSSION TITLE
TX Equalizer

### DISCUSSION THREAD

**Discussion by syracusepro** [ramdor/Thetis#361] [Ideas]
It is possible to expand the TX equalizer from 10 bands to 20? It will be nice to add a more range of frequencies to EQ difficult natural voice audio. Thanks.

**Comment by w-u-2-o** [comment]
The transmit audio facilities in Thetis should be more than adequate for
any raw material. If you are having difficulty achieving results you like
you might try asking on the Apache Forum site for some advice.
